client/components/Events/EventItem.tsx

```diff
@@ -219,8 +219,12 @@ class EventItemComponent extends React.Component<IProps, IState> {
                 >
                     <Row>
                         <span className="sd-overflow-ellipsis sd-list-item--element-grow">
-                            {renderFields(get(listFields, 'event.primary_fields',
-                                EVENTS.LIST.PRIMARY_FIELDS), item, {}, language)}
+                            {renderFields(
+                                get(listFields, 'event.primary_fields', EVENTS.LIST.PRIMARY_FIELDS),
```

Comment:
Ideally we should be replacing lodash functions with typescript syntax, where applicable.

Reply:
This lad is quick!